lib/grape-route-helpers/decorated_route.rb

```diff
@@ -1,16 +1,15 @@
 module GrapeRouteHelpers
-  # wrapper around Grape::Route that adds a helper method
+  # wrapper around Grape::Route#@options that adds a helper method
   class DecoratedRoute
-    attr_reader :route, :helper_names, :helper_arguments,
+    attr_reader :helper_names, :helper_arguments,
                 :extension, :route_options
 
     def self.sanitize_method_name(string)
       string.gsub(/\W|^[0-9]/, '_')
     end
 
-    def initialize(route)
-      @route = route
-      @route_options = route.instance_variable_get(:@options)
+    def initialize(route_options)
```

Comment:
I bet this class must be initialized with `route` but not `route_options`.
Using `route_options` leads to a cleaner code but violates naming convention.